flask_resources/parsers/headers.py

```diff
@@ -1,29 +1,31 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2020 CERN.
-# Copyright (C) 2020 Northwestern University.
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020-2021 Northwestern University.
 #
 # Flask-Resources is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Library for easily implementing REST APIs."""
+"""Header parser."""
 
+import marshmallow as ma
 from flask import request
 
 from .base import RequestParser, request_parser_decorator
 
 
-def _raw_args(*args, **kwargs):
-    return {**request.headers}
-
-
 class HeadersParser(RequestParser):
     """Headers parser."""
 
-    location = "headers"
-    raw_args_fn = _raw_args
+    def load_data(self):
```

Comment:
discussion: I think HeadersParser is a parser that should allow everything by default. Authentication headers, e-tag headers, cache.... there are a lot of headers we want without having to list them all, right?

Reply:
Discussed during standup: for now stay with a restricted list, and if later have examples we could more easily open up and include all unknown values.